### DISCUSSION TITLE
Solar system

### DISCUSSION THREAD

**Discussion by Bart-van-Gorp** [Madelena/Metrology-for-Hass#68] [General]
<img width="1257" alt="177698759-1770e1a0-eff4-4f3a-aee3-4643733c450c" src="https://github.com/Madelena/Metrology-for-Hass/assets/6367588/fbfbce70-64ee-4ca5-aef8-4a1e25c1fa44">
I just saw this image on the home assistant live show. Does someone know what integrations are used to build this?

I'd like to make the exact dashboard for our home.

**Comment by Tobe2d** [comment]
+1 would love to know how this done.
Hope you could point out how to do it please

**Comment by Madelena** [comment]
You can find the source of this dashboard view on my public HA config repo here:
https://github.com/Madelena/hass-config-public/blob/main/ui/views/astrometrics.yaml

Live long and prosper, beltalowda! 😊🖖
